.github/workflows/release_linux.yml

```diff
@@ -20,10 +21,22 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - os: ubuntu-22.04
-            pretty: "Ubuntu 22.04"
-          - os: ubuntu-24.04
-            pretty: "Ubuntu 24.04"
+        - os: ubuntu-22.04
+          pretty: "Ubuntu 22.04"
+          arch: x86_64
+          python-arch: x64
+        - os: ubuntu-24.04
+          pretty: "Ubuntu 24.04"
+          arch: x86_64
+          python-arch: x64
+        - os: ubuntu-22.04-arm
+          pretty: "Ubuntu 22.04 arm64"
+          arch: arm64
+          python-arch: arm64
+        - os: ubuntu-24.04-arm
+          pretty: "Ubuntu 24.04 arm64"
+          arch: arm64
+          python-arch: arm64
```

Comment:
Add some arm runners. Now I need to also not use hardcoded x64 or x86_64 values, so add `arch` and `python-arch` (like on the mac release yml)

Reply:
Got it.  This is good stuff.  I was kinda surprised to see 22.04 added in here, but hey it's fine for a while.